src/burn/engine/bitsengine.cpp

```diff
@@ -439,11 +442,19 @@ static HRESULT CreateJob(
     hr = pJob->SetNotifyFlags(BG_NOTIFY_JOB_TRANSFERRED | BG_NOTIFY_JOB_ERROR | BG_NOTIFY_JOB_MODIFICATION);
     ExitOnFailure(hr, "Failed to set notification flags for BITS job.");
 
-    hr = pJob->SetNoProgressTimeout(BITSENGINE_NO_PROGRESS_TIMEOUT); // use 2 minutes since default is 14 days.
-    ExitOnFailure(hr, "Failed to set progress timeout.");
+    if (fBackgroundDownload)
+    {
+        hr = pJob->SetPriority(BG_JOB_PRIORITY_LOW);
```

Comment:
Yes, and the bundles we're doing this on are big - Windows Phone and Android emulators. This went with the async feature so with that it didn't matter (to answer @rseanhall's question about why the PR from last spring was closed, since OSS didn't want async execution). Without async, backgrounding a BITS job may not make sense.